setup.py

```diff
@@ -759,21 +759,21 @@ def printException_loggerwarn():
 "Press any key when you are done and ready.", sep="\n")
 input()
 print("Please input your API key below.")
-apikey_input = input("Input here: ")
+apikey_input = str(input("Input here: ")).replace(" ","").replace("\n","").replace("\t","")
 logger.debug("apikey_input: %s" % apikey_input)
-print("", "Just to confirm, the API key you gave me was: " + apikey_input
-      + ".", sep="\n")
+print("", "Just to confirm, the API key you gave me was: '" + apikey_input
+      + "'", sep="\n")
 print("Please double check your input, and confirm in the dialogue below.")
 apikey_confirm = input("Is the API key right? Yes or no: ").lower()
 logger.debug("apikey_confirm: %s" % apikey_confirm)
 if apikey_confirm == "no":
     while True:
         logger.debug("User now re-entering key...")
         print("","Please input your API key below.", sep="\n")
-        apikey_input = input("Input here: ")
+        apikey_input = str(input("Input here: ")).replace(" ","").replace("\n","").replace("\t","")
         logger.debug("apikey_input: %s" % apikey_input)
-        print("Just to confirm, the API key you gave me was: " + apikey_input
-              + ".")
+        print("Just to confirm, the API key you gave me was: '" + apikey_input
+              + "'")

```

Comment:
Same goes here, the period is part of the dialogue. I think we only need a replace statement and not necessarily change to the setup dialogue.